tools/clang/unittests/SPIRV/CodeGenSpirvTest.cpp

```diff
@@ -2192,4 +2192,10 @@ TEST_F(FileTest, MeshShadingNVAmplificationError4) {
   runFileTest("meshshading.nv.error3.amplification.hlsl", Expect::Failure);
 }
 
+// Test Vulkan1.2 target environment
+TEST_F(FileTest, TextureLoadVk1p2) {
```

Comment:
Since the purpose of the test is **not** Texture loading, we should change the test name and file name.

For example 
Test name: `Vk1p2EntryPoint`
File name: `vk.1p2.entry-point.hlsl`